codespell_lib/data/dictionary.txt

```diff
@@ -2061,6 +2061,9 @@ autimatically->automatically
 autmatically->automatically
 auto-dependancies->auto-dependencies
 auto-destrcut->auto-destruct
+auto-detet->auto-detect
+auto-deteted->auto-detected
```

Comment:
At least this and detection should also suggest deletion variants.

Reply:
please elaborate

Reply:
Sorry `auto-deteted->auto-detected, auto-deleted,`